### ISSUE TITLE
Whitelist bundle for AEM 6.3

### ISSUE THREAD

**Issue by mickleroy** [shinesolutions/aem-password-reset#5]
In AEM 6.3, any bundles using `getAdministrativeResourceResolver` need to be whitelisted via `org.apache.sling.jcr.base.internal.LoginAdminWhitelist.fragment`

**Comment by ahujaankur** [comment]
1. Update the code with whitelisting the bundle inside Shine code base as OSGI Config.
2. Removed deprecated APIs and updated the code with latest dependencies and APIs.

**Comment by mickleroy** [comment]
Fixed in release 1.1.0

**Comment by cliffano** [comment]
Confirming that the error stack trace no longer shows up, and after the fix in 1.1.0, the success log output looks like this:
```10.07.2018 13:54:30.857 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for admin
10.07.2018 13:54:30.858 *INFO* [FelixStartLevel] com.adobe.granite.security.user.internal.audit.AuditAuthorizableAction Password for User 'orchestrator' was changed
10.07.2018 13:54:30.862 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for orchestrator
10.07.2018 13:54:30.863 *INFO* [FelixStartLevel] com.adobe.granite.security.user.internal.audit.AuditAuthorizableAction Password for User 'replicator' was changed
10.07.2018 13:54:30.871 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for replicator
10.07.2018 13:54:30.871 *INFO* [FelixStartLevel] com.adobe.granite.security.user.internal.audit.AuditAuthorizableAction Password for User 'deployer' was changed
10.07.2018 13:54:30.890 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for deployer
10.07.2018 13:54:30.890 *INFO* [FelixStartLevel] com.adobe.granite.security.user.internal.audit.AuditAuthorizableAction Password for User 'exporter' was changed
10.07.2018 13:54:30.898 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for exporter
10.07.2018 13:54:30.898 *INFO* [FelixStartLevel] com.adobe.granite.security.user.internal.audit.AuditAuthorizableAction Password for User 'importer' was changed
10.07.2018 13:54:30.907 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset.Activator Changed the password for importer
10.07.2018 13:54:30.908 *INFO* [FelixStartLevel] com.shinesolutions.aem.passwordreset BundleEvent STARTED
```
